packages/quantic/force-app/examples/main/lwc/exampleQuanticResultQuickview/exampleQuanticResultQuickview.js

```diff
@@ -1,68 +1,76 @@
-import { registerComponentForInit } from 'c/quanticHeadlessLoader';
+import {registerComponentForInit} from 'c/quanticHeadlessLoader';
 import {api, LightningElement, track} from 'lwc';
 
 export default class ExampleQuanticResultQuickview extends LightningElement {
-    @api engineId = 'quantic-result-quickview-engine';
-    @track config = {};
-    isConfigured = false;
+  @api engineId = 'quantic-result-quickview-engine';
+  @track config = {};
+  isConfigured = false;
 
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+  }
 
-    connectedCallback() {
-        registerComponentForInit(this, this.engineId);
-    }
+  result = {
```

Comment:
If you wanted, I'm sure you can create an js file next to this component and define all these object constants in that different file so they get exported and you can import them in this file.
(If you wanted to split it so this component doesn't declare a bunch of constants inline of the code)